CONDUCT.md

```diff
@@ -1,4 +1,4 @@
-# Code of Conduct
+# Contributor Covenant Code of Conduct
```

Comment:
My thought is you don't need to put Contributor Covenant on this as we stated below that we adapted from Contributor Covenant.
Also this one is our Code of Conduct, not the Contributor Covenant Code of Conduct.

Reply:
Sorry, I just compare with the example one from Tiff and see that yours is like the example one.
I'll change my review and approve now.